thirdparty/faiss/faiss/impl/index_read.cpp

```diff
@@ -63,9 +63,17 @@ namespace faiss {
 static void read_index_header(Index* idx, IOReader* f) {
     READ1(idx->d);
     READ1(idx->ntotal);
+    READ1(idx->is_cosine);
+
+    uint8_t dummy8;
+    READ1(dummy8);
+    uint16_t dummy16;
```

Comment:
8(used for cosine) + 8 + 16 + 32 (rest padding) = 64 ? Pls add some comments here or just use 64 bit to save the `is_cosine`

Reply:
yes